services/common/src/interfaces/activity/notificationDocumentMetadata.interface.ts

```diff
@@ -3,7 +3,7 @@ import {
   INotificationDocumentPermit,
   INotificationDocumentProject,
   INotificationDocumentMineTailingsStorageFacility,
-} from "@/index";
```

Comment:
Oh good!  I actually just came across a few of these in my current work and thought it didn't look quite right (although wasn't showing any errors)

Reply:
In the previous setup `@/index` worked fine, but now it'll have to be `@mds/common` going forwards